<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Medico_cadastro</class>
 <widget class="QMainWindow" name="Medico_cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="botaoIncio">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>In√≠cio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoAtendimentos">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Atendimentos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoCadastros">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(144, 144, 144);
background-color: rgb(176, 251, 255);
border-color: rgb(48, 245, 255);
</string>
        </property>
        <property name="text">
         <string>Cadastro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoSair">
        <property name="styleSheet">
         <string notr="true">QPushButton { 
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(48, 245, 255);
}

QPushButton:pressed { 
color: rgb(68, 68, 68);
background-color: rgb(170, 255, 255);
border-color: rgb(48, 245, 255);
}</string>
        </property>
        <property name="text">
         <string>Sair</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_edite">
       <attribute name="title">
        <string>Edite seu cadastro</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_edite">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Edite o seu cadastro:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_id">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_num_id">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_nome">
             <property name="text">
              <string>Nome completo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_nome"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_username">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_username"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_senha">
             <property name="text">
              <string>Senha:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_senha"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="botaoCancelar">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 126, 126);
background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botaoAtualizar">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(7, 255, 226);
border-color: rgb(48, 245, 255);
</string>
             </property>
             <property name="text">
              <string>Atualizar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
